sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md

```diff
@@ -36,6 +36,7 @@ All of these samples need the endpoint to your Form Recognizer resource ([instru
 |[TrainUnlabeledModel][train_unlabeled_model] and [TrainUnlabeledModelAsync][train_unlabeled_model_async]|Train a custom model with unlabeled data|
 |[ManageCustomModels][manage_custom_models] and [ManageCustomModelsAsync][manage_custom_models_async]|Manage the custom models in your account|
 |[CopyModel][copy_model] and [CopyModelAsync][copy_model_async]|Copy custom model from one Form Recognizer resource to another|
+|[CreateComposedModel][create_composed_model] and [CreateComposedModelAsync][create_composed_model_async]|Creates a composed model from a collection of existing trained models with labels|
```

Comment:
For some reason all links are broken

Reply:
Yeah, they were using the commit ID, will update these in an upcoming PR